app/ui/browser/views/page/status.jsx

```diff
@@ -24,36 +23,29 @@ const STATUS_STYLE = Style.registerStyle({
   zIndex: 1,
   padding: '2px 10px',
   borderTopRightRadius: '3px',
-  border: '1px solid #eee',
-  backgroundColor: '#fff',
-  color: '#555',
+  border: '1px solid var(--theme-splitter-color)',
+  borderBottomWidth: 0,
+  borderLeftWidth: 0,
+  backgroundColor: 'var(--theme-body-color)',
+  color: 'var(--theme-content-color)',
 });
 
 /**
- * The old status bar at the bottom of the page
+ * A status bar at the bottom of the browser UI.
  */
 
-const Status = ({ page, statusText }) => {
-  let contents;
-
-  if (statusText) {
-    contents = statusText;
-  } else if (page.isLoading) {
-    contents = 'Loading...';
-  }
-
+export function Status({ statusText }) {
   return (
     <div className={STATUS_STYLE}
-      hidden={!contents}>
-      {contents}
+      hidden={!statusText}>
+      {statusText}
     </div>
   );
-};
+}
 
 Status.displayName = 'Status';
 
 Status.propTypes = {
-  page: PropTypes.object.isRequired,
```

Comment:
We don't use it anymore. We still pass it in somewhere, but the only reason it was used was because we cared about the `isLoading` property on a page, which doesn't exist anymore. We use the `getStatusText` selector now for status texts.